### PR TITLE
[16.0][MIG] stock_picking_send_by_mail: Migration to version 16.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration to v16
- Supersedes https://github.com/OCA/stock-logistics-workflow/pull/1272
- Fix tests added on PR https://github.com/OCA/stock-logistics-workflow/pull/1272

TT46603

@Tecnativa

@pedrobaeza @pilarvargas-tecnativa

### CODE REVIEW

File: stock_picking_send_by_mail/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Stock Picking by Mail",
     "summary": "Send stock picking by email",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.1",
```
Comment (pedrobaeza) [inline stock_picking_send_by_mail/__manifest__.py]: The version should be `16.0.1.0.0` and must be changed in the migration commit, not in the pre-commit stuff.

File: stock_picking_send_by_mail/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+==========================
+Stock picking send by mail
+==========================
+
```
Comment (chienandalu) [inline stock_picking_send_by_mail/readme/DESCRIPTION.rst]:
```suggestion
```

**Review by chienandalu — CHANGES_REQUESTED** [review]

File: stock_picking_send_by_mail/__manifest__.py (hunk 1 above)
Comment (carolinafernandez-tecnativa) [reply stock_picking_send_by_mail/__manifest__.py]: could you please check it now?

**Review by pedrobaeza — APPROVED** [review]
I don't see your name in CONTRIBUTORS though.

**Comment by carolinafernandez-tecnativa** [comment]
> I don't see your name in CONTRIBUTORS though.

Done!

**Review by pilarvargas-tecnativa — APPROVED** [review]
Revised functionality, thanks

**Pull request merged**